models/DataObject/ClassDefinition.php

```diff
@@ -466,7 +466,9 @@ public function save($saveDefinitionFile = true)
             $this->dispatchEvent(new ClassDefinitionEvent($this), DataObjectClassDefinitionEvents::PRE_UPDATE);
         }
 
-        $this->setModificationDate(time());
```

Comment:
@jremmurd instead of using ``time()``, the timestamp should be extracted from the class definition file.

Reply:
This is a line which is removed. See line 116 within the source branch in models/DataObject/ClassDefinition/Dao.php